### DISCUSSION TITLE
Header filter like excel

### DISCUSSION THREAD

**Discussion by good-ghost** [ghiscoding/slickgrid-universal#1185] [Q&A] [answered]
Many thanks for this Universal repo for Salesforce.

BTW, how can I use extra Slickgrid plugin to support Header filter as below?
It is replacement of Slick.headermenu.js plugin.

[Header filter example](https://danny-sg.github.io/slickgrid-spreadsheet-plugins/examples/example-1-everything.htm)

Thanks,

**Comment by ghiscoding** [comment, marked as answer]
Read the [Styling CSS/SASS - Wiki](https://github.com/ghiscoding/slickgrid-universal/wiki/Styling#using-built-in-themes), you can't use SASS in Salesforce but you can use CSS variables to change the styling and add border to the menu maybe show it all the time, etc... The menu content will never look like that other fork, my project will never include the list of values to filter against since we only include a list of commands not a list of values, if you ever want this, you will have to create your own but that might not be possible in Salesforce
